nf_core/launch.py

```diff
@@ -208,6 +223,32 @@ def get_pipeline_schema(self):
                 log.error("Could not build pipeline schema: {}".format(e))
                 return False
 
+    def try_fetch_release_tags(self):
+        """Tries to fetch tag names of pipeline releases from github
+
+        Returns:
+            release_tags (list[str]): Returns list of release tags
+
+        Raises:
+            LookupError, if no releases were found
+        """
+        # Fetch releases from github api
+        releases_url = "https://api.github.com/repos/nf-core/{}/releases".format(self.pipeline)
+        response = requests.get(releases_url)
+        if not response.ok:
+            log.error(f"Unable to find any release tags for {self.pipeline}. Will try to continue launch.")
+            raise LookupError
+
+        # Filter out the release tags and sort them
+        release_tags = map(lambda release: release.get("tag_name", None), response.json())
+        release_tags = filter(lambda tag: tag != None, release_tags)
+        release_tags = list(release_tags)
+        if len(release_tags) == 0:
+            log.error(f"Unable to find any release tags for {self.pipeline}. Will try to continue launch.")
+            raise LookupError
+        release_tags = sorted(release_tags, key=lambda tag: tuple(tag.split(".")), reverse=True)
```

Comment:
```suggestion
        release_tags = sorted(release_tags, key=lambda tag: tuple(
            [int(x) if x.isdigit() else 0 for x in tag.split(".")]), reverse=True)
```
or similar? Not great, but currently

```
>>> release_tags=['1', '10', '12', '20', '270', '102', '1000']
>>> sorted(release_tags, key=lambda tag: tuple(tag.split(".")), reverse=True)
['270', '20', '12', '102', '1000', '10', '1']
>>> 
```

Reply:
(`sorted` should be stable, so with the suggested code "non-numeric" tag parts should not cause reordering (`master`, `dev`), but e.g.`10.2alfa` would come before `10.1` as well as `10.3`.)

Reply:
Now sorts by release publication date instead, which I think is the easiest and probably best method still.